conf/messages.en

```diff
@@ -711,6 +711,47 @@ assets.removeYesNo.title = Are you sure you want to remove the asset?
 assets.removeYesNo.heading = Are you sure you want to remove {0}?
 assets.removeYesNo.error.required = Select yes if you want to remove the asset
 
+assetAddressUkYesNo.title = Is the business asset address in the UK?
+assetAddressUkYesNo.heading = Is {0} address in the UK?
+assetAddressUkYesNo.checkYourAnswersLabel = Is {0} address in the UK?
+assetAddressUkYesNo.error.required = Select yes if the business's address in the UK
+
+assetDescription.title = What is the description for the business asset?
+assetDescription.heading = What is the description for {0}?
+assetDescription.checkYourAnswersLabel = What is the description for {0}?
+assetDescription.hint = This must be 70 characters or less
+assetDescription.error.required = Enter a description of the business
+assetDescription.error.invalid = The description of the beneficiary must only include letters a to z, numbers, ampersands (&), apostrophes, forward slashes, full stops, hyphens, round brackets and spaces
+assetDescription.error.length = The description of the beneficiary must be 70 characters or less
+
+assetInternationalAddress.title = What is the business asset address?
+assetInternationalAddress.heading = What is {0}’s address?
+assetInternationalAddress.checkYourAnswersLabel = What is {0}’s address?
+
+assetName.title = What is the business’s name?
+assetName.heading = What is the business’s name?
+assetName.checkYourAnswersLabel = What is the business’s name?
+assetName.error.required = Enter the business’s name
+assetName.error.length = Company’s name cannot be more than 53 characters
+assetName.error.format = Company’s name must only include letters a to z, numbers, ampersands (&), apostrophes, forward slashes, full stops, hyphens, round brackets and spaces
+
+assetUkAddress.title = What is the business asset address?
```

Comment:
Should be "What is the business asset’s address?" since the heading uses the ’s format